app/assets/javascripts/backbone/views/timeline_select_steps_view.coffee

```diff
@@ -9,10 +9,9 @@ class MS.Views.TimelineSelectStepsView extends Backbone.View
     'click #uncheck-all-ctrl': 'uncheckAll'
     'click #check-completed-ctrl': 'checkCompleted'
     'click #check-incomplete-ctrl': 'checkIncomplete'
- 
+
   checkAll: (e) ->
-    $control = $(e.currentTarget)
-    $inputs = $control.closest('.timeline').find('.select-step')
+    $inputs = @$el.find('.select-step')
```

Comment:
It has been awhile since I have done coffeescript+backbone. I am not sure what `this` is bound to in this context. Not sure if I could also just shorten this to `this.find('.select-step')` or even `find('.select-step')`?